project.clj

```diff
@@ -10,16 +10,21 @@
 
   :description "SSL certificate management on the JVM."
 
+  :min-lein-version "2.7.1"
+
+  :parent-project {:coords [puppetlabs/clj-parent "0.2.5"]
+                   :inherit [:managed-dependencies]}
+
   ;; Abort when version ranges or version conflicts are detected in
   ;; dependencies. Also supports :warn to simply emit warnings.
-  ;; requires lein 2.2.0+.
   :pedantic? :abort
 
-  :dependencies [[org.clojure/tools.logging "0.2.6" :exclusions [org.clojure/clojure]]
+  :dependencies [[org.clojure/tools.logging]
                  [org.bouncycastle/bcpkix-jdk15on "1.55"]
-                 [commons-codec "1.9"]
-                 [clj-time "0.7.0"]
```

Comment:
Beware...when I added this `clj-time` dependency, we specifically needed something only available in `0.7.0`.

There were other Clojure projects (PuppetDB I think?) that had older `clj-time` dependencies, like `0.5.0`, which would result in an error at runtime.

Maybe this situation doesn't exist anymore, or it's gotten better (i.e. every project is on a new-enough `clj-time`), but something to be aware of.